File: lua/fizz.lua
local fizz = {
    ["3"] = "fizz",
    ["5"] = "buzz",
}

for i = 1, 100, 1 do
    local out = ""
    for k, v in pairs(fizz) do
        if i % tonumber(k) == 0 then
            out = out .. v
        end
    end
    if out == "" then
        out = tostring(i)
    end
    print(out)
end

File: python/fizz.py
"""Fizzbuzz"""


def main():
    """
    FizzBuzz
    """
    fizzbuzz_map = {"3": "fizz", "5": "buzz"}
    for i in range(1, 101):
        output = ""
        for key, val in fizzbuzz_map.items():
            if i % int(key) == 0:
                output += val
        if len(output) == 0:
            output = str(i)
        print(output)


if __name__ == "__main__":
    main()
